src/constants/css.ts

```diff
@@ -1,3 +1,4 @@
+// tslint:disable
```

Comment:
It better to disable the concerned rule instead of disabling everything: 
`// tslint:disable:max-line-length`